README.md

```diff
@@ -47,7 +48,7 @@ youngs_modulus = 42.0
 poissons_ratio = 0.3
 
 mesh = df.mesh.create_unit_cube(MPI.COMM_WORLD, 2, 2, 2)
-V = df.fem.VectorFunctionSpace(mesh, ("CG", 1))
+V = df.fem.functionspace(mesh, ("CG", 2, (2,)))
```

Comment:
You changed the order of the function space from the old to the new version, right?
Just for my understanding, what's the second 2?